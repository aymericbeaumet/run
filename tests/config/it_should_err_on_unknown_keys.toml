__unknown_key__ = true
