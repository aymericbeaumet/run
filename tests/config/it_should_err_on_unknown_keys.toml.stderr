Error: TOML parse error at line 1, column 1
  |
1 | __unknown_key__ = true
  | ^^^^^^^^^^^^^^^
unknown field `__unknown_key__`, expected one of `run`, `mode`, `openai`, `prefix`, `tmux`, `workdir`

